bundles/org.smarthomej.binding.viessmann/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -198,12 +198,19 @@
 	</channel-type>
 
 	<channel-type id="type-liter">
-		<item-type>Number:Time</item-type>
+		<item-type>Number:Volume</item-type>
```

Comment:
Can you add upgrade instructions for this? Otherwise the user has to delete and re-add. the thing.

Reply:
I added your requested changes